retrofit/src/main/java/retrofit/Call.java

```diff
@@ -33,4 +33,5 @@
   void enqueue(Callback<T> callback);
   void cancel();
   Call<T> clone();
+  String infoForException();
```

Comment:
Yeah, me too. 

`toString()` will play really nice with IntelliJ's 14 inline debugger comments feature! And it looks like a good choice for cases when user accidentally includes `call` into exception message.

Will rename.